contracts/tokens/SecurityToken.sol

```diff
@@ -740,16 +734,23 @@ contract SecurityToken is StandardToken, DetailedERC20, ReentrancyGuard, Registr
         require(_burn(msg.sender, _value, _data), "Burn invalid");
     }
 
+    function _burnFrom(address _from, uint256 _value, bytes _data) internal returns(bool) {
```

Comment:
Why do we need this? Isn't our `_burn` same as this?

Reply:
It invokes _burnFrom(_from, _value), which has an additional line: 
_allowed[account][msg.sender] = _allowed[account][msg.sender].sub(
      value);

Reply:
Oh, the allowances.
@adamdossa Do we need to separate `_burn` and `_burnFrom` when the module/owner can use either one?

Reply:
@maxsam4 both are different `_burn` and `_burnFrom` so we need both of them.

Reply:
I don't know what was I thinking when I posted that. Sorry. Probably I was under the impression that module can call _burn and pass anyone else's address. Clearly, they can't.
This reminds me, I'll check if burn is a public function in OZ.